src/script/target/getWebpackConfiguration.js

```diff
@@ -0,0 +1,3 @@
+const { getConfiguration } = require('./webpack')
+
+module.exports = state => getConfiguration(state)
```

Comment:
Too much files to get `getConfiguration` function :
`target/index.js` calls `target/getWebpackConfiguration` calls `target/webpack/index.js` calls `target/webpack/getConfiguration`